Primo.js

```diff
@@ -213,7 +213,13 @@ function importPNX(text, url) {
 	
 	var itemType = ZU.xpathText(doc, '//display/type');
 	if(!itemType) {
-		throw new Error('Could not locate item type');
+		if ( ZU.xpathText(doc, '//facets/rsrctype') ) {
+			var itemType = ZU.xpathText(doc, '//facets/rsrctype');
+		} else if ( ZU.xpathText(doc, '//search/rsrctype') ) {
+			var itemType = ZU.xpathText(doc, '//search/rsrctype');
```

Comment:
using `ristype` makes sense, but would require a fair amount of testing across instances, since the current item type detection if heavily tested and I don't trust PNX/Primo to perfectly follow the RIS item types, so I'm leaving this for another time. 
Implemented the simplified syntax, though. Good point there.